src/NHibernate.Test/Async/DynamicProxyTests/PeVerifier.cs

```diff
@@ -0,0 +1,51 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by AsyncGenerator.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using NUnit.Framework;
+
+namespace NHibernate.Test.DynamicProxyTests
+{
+	using System.Threading.Tasks;
+	
+	/// <content>
+	/// Contains generated async methods
+	/// </content>
+
+	public partial class PeVerifier
```

Comment:
This is also strange, I excluded the `DeadlockHelper` but the `PeVerifier` and `PeVerifyFixture` were generated. In order to exclude the `StreamReader.ReadToEnd` method add the following setting to the test project (must be inside `analyzation` setting):
```
ignoreSearchForAsyncCounterparts:
    - name: ReadToEnd
```
with this setting, the `PeVerifier` and `PeVerifyFixture` won't be generated as they were generated only because of the async version of the `StreamReader.ReadToEnd` method.

Reply:
The main cause why `PeVerifier` is generated is because of [this](https://github.com/nhibernate/nhibernate-core/blob/ad4c2ef101cbf8ba798220973f4f78dd795e0896/src/NHibernate.Test/Component/Basic/ComponentTest.cs#L35) line. As `ComponentTest` is a fixture all its methods bodies will get scanned for async counterparts. When the `ReadToEnd` is found the generator checks all its usages and there is where `PeVerifier` is found.